templates/readme/README.md

```diff
@@ -46,31 +46,32 @@ On the backend, use `manifest.json` to look up which files to load and serve.
 
 ### React components
 
-Put React components in `source/components`. It is recommended to have each component in a separate folder, containing a `jsx` file, a `scss` file and an `index.js` file.
-
-If the folder and the `jsx` file have the same name, the component will be included in the generated `app.components.js` file which can be used to render components on a backend (like ReactJS.NET).
+Put React components in `source/components`. If the folder and the `jsx` file have the same name, the component will be included in the generated `app.components.js` file which can be used to render components on a backend (like ReactJS.NET).
 
 When building a static site, `app.components.js` is not generated and `server.js` is not output as part of the build.
 
-### Aliases
+### Import paths
 
-By default, two aliases are included in `webpack.config`:
+`webpack.config` specifies two aliases (under `resolve.alias`) that make importing files easier:
 
 - `components` which resolves to `source/components`
 - `js` which resolves to `source/js`
 
-These aliases allow you to import like this from any `js`/`jsx` file:
+Also, `directory-named-webpack-plugin` is used to automatically import files that have the same name as their containing folder. Examples:
 
-```
-import SomeComponent from 'components/some-component';
-import someScript from 'js/some-script';
+```js
+// <project root>/source/components/some-component/some-component.jsx
+import SomeComponent from "components/some-component";
+
+// <project root>/source/js/some-script.js
+import someScript from "js/some-script";
```

Comment:
`./source/js` is not included as a path for the plugin. Does this work?

Reply:
It should, yes. As I understand the plugin, essentially what it does is to modify module resolution. Instead of `folder/index.js || folder.js || node_modules/folder` it now looks for `folder/index.js || folder/folder.js || folder.js || node_modules/folder` (when importing "folder").

Reply:
Lol, commented on the wrong thing :P

Reply:
Yes, the above should work, but `js/some-script/some-script.js` can't be imported as `js/some-script` right now. I've only included component folders as this pattern seems most useful there

Reply:
@pellebjerkestrand I updated the readme. Hopefully it makes this a bit clearer